charts/k8ssandra/values.yaml

```diff
@@ -156,30 +104,87 @@ cass-operator:
   enabled: true
 
 kube-prometheus-stack:
-  alertmanager:
+  # Disable default service monitors
+  coreDns:
     enabled: false
-
-  kubeStateMetrics:
+  kubeApiServer:
     enabled: false
-
-  grafana:
+  kubeControllerManager:
     enabled: false
-
-  nodeExporter:
+  kubeDns:
     enabled: false
-
-  kubeletService:
+  kubeEtcd:
     enabled: false
-
-  kubeControllerManager:
+  kubeProxy:
+    enabled: false
+  kubeScheduler:
+    enabled: false
+  kubeStateMetrics:
     enabled: false
-
   kubelet:
     enabled: false
+  nodeExporter:
+    enabled: false
 
-  kubeApiServer:
+  alertmanager:
     enabled: false
+    serviceMonitor:
+      selfMonitor: false
+
+  # Locks Prometheus operator to this namespace and kube-system
+  prometheusOperator:
+    namespaces:
+      releaseNamespace: true
+      additional: []
+
+    # Disable monitoring of operator
+    serviceMonitor:
+      selfMonitor: false
 
   prometheus:
-    enabled: false
```

Comment:
Don't we need the `enabled` flag for Prometheus?

Reply:
It defaults to true so I didn't include it here. Users may still override this parameter in their local values.yaml if they want.

Reply:
I think it is a good idea to still include the `enabled` flag. It makes things more explicit, albeit a bit more verbose. It also serves as documentation for the user.

Reply:
Sure, I'm happy to bring all these parameters into `values.yaml`.